INTERIOR/PinInteriorInMemory.md

```diff
@@ -9,6 +9,22 @@ aliases: ["0x2CA429C029CCF247","_LOAD_INTERIOR"]
 void PIN_INTERIOR_IN_MEMORY(int interior);
 ```
 
+Instructs the interior system to load the specified interior into memory.
+
+```
+NativeDB Introduced: v323
+```
+
 ## Parameters
-* **interior**: 
+* **interior**: The index of the interior.
 
+## Examples
+```lua
+CreateThread(function()
+    local interiorID = GetInteriorAtCoords(976.6364, 70.29476, 115.1641)
+    PinInteriorInMemory(interiorID)
+    repeat
```

Comment:
Perhaps it would be best do do something below this while loop to show why you should wait?

Reply:
You mean like this?

```lua
CreateThread(function()
    -- Coordinates of the interior
    local interiorID = GetInteriorAtCoords(976.6364, 70.29476, 115.1641)
    
    -- Load the interior into memory
    PinInteriorInMemory(interiorID)
    
    -- Wait until the interior is ready
    repeat
        Wait(0)
    until IsInteriorReady(interiorID)
    
    -- Teleport the player to the interior
    local playerPed = PlayerPedId()
    SetEntityCoords(playerPed, 976.6364, 70.29476, 115.1641)
end)
```